src/javascript/binary/base/page.js

```diff
@@ -523,6 +523,27 @@ Header.prototype = {
             BinarySocket.send({"logout": "1"});
         });
     },
+    
+    validate_cookies: function(){
+        function is_valid_id(id){
+            return /^(MX|MF|VRTC|MLT|CR|FOG|VRTJ|JP)[0-9]+$/i.test(id);
```

Comment:
i don't these broker codes should be in frontend, we should have this check in backend in my opinion. Even if we go with this then remove FOG as will never be there